test/widgets/subscription_list_test.dart

```diff
@@ -190,4 +202,47 @@ void main() {
     check(tester.widget<UnreadCountBadge>(find.byType(UnreadCountBadge)).backgroundColor)
       .equals(swatch);
   });
+
+  testWidgets('muted streams are displayed as faded', (tester) async {
+
+    testOpacityForStreamAndBadge(String streamName, double opacity) {
```

Comment:
nit: names starting with "test" are reserved for helpers that substitute for `test`, so that create a new test case (by calling `test`, directly or indirectly).

For a helper like this, say "check".